www/demos/demos.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: prose.vto
+templateEngine: vto
+url: /demos/
+renderOrder: 1
+subtitle: missing.css
+title: Demos
+crumbs:
+ - url: /
+   title: Home
+---
+
+<h3>
+<sub-title>missing.css<v-h>: </v-h></sub-title>
+Full Page Examples
+</h3>
+<p>
+The following pages demonstrate functionality that cannot fit in the docs.
+<ul class="list-of-links flow-gap textcolumns">
+{{ for demo of search.pages("url^=/demos/") |>filter((demo) => demo.page !== page & !demo.url.includes("/apg/")) }}
+<li><a href={{ demo.url }}>{{ demo.title }}</a>
+{{ /for }}
+</ul>
+
+<h3>
+<sub-title>missing.css<v-h>: </v-h></sub-title>
```

Comment:
I'm not 100% on `<sub-title>missing.css</sub-title>` here for the `<h3>` tags, but I wanted to add some context that the links contain our implementations of the APG patterns, not just links to the APG site.

Reply:
We voted to remove them